src/ontology/mondo-edit.obo

```diff
@@ -244551,12 +244550,13 @@ subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "ADEAF" EXACT ABBREVIATION [Orphanet:101046]
 synonym: "ADLTE" EXACT ABBREVIATION [Orphanet:101046]
+synonym: "adolescent/adult onset autosomal dominant epilepsy with auditory features" RELATED [https://www.epilepsydiagnosis.org/syndrome/adeaf-overview.html]
```

Comment:
Is this right? This synonym is the term label for the term merged, and it is "related" synonym.
Also, the synonym below, "autosomal dominant partial/lateral temporal epilepsy with auditory features" (which does seem not exact because of the "partial/lateral" is exact.

Reply:
‘adolescent/adult onset autosomal dominant epilepsy with auditory features’ was formerly used in ILAE but this naming convention seems to be no longer used in ILAE. (Note - ILAE says: This syndrome typically has onset between 10-30 years (range 0.5-54 years).)

Should it be exact (or narrow) and historic (DEPRECATED)?

Reply:
Let's add "historic". 
If it was the name for this specific disease, then let's have the synonym scope as exact.

Reply:
updated 🎉